extensions/wikia/Forum/RelatedForumDiscussionController.class.php

```diff
@@ -42,28 +42,28 @@ public function checkData() {
 		$articleId = $this->getVal( 'articleId' );
 		$title = Title::newFromId( $articleId );
 		if ( empty( $articleId ) || empty( $title ) ) {
-			$this->replace = false;
-			$this->articleId = $articleId;
+			$this->response->setVal('replace', false);
+			$this->response->setVal('articleId', $articleId);
 			return;
 		}
 
 		$messages = RelatedForumDiscussionController::getData( $articleId );
 
 		$timediff = time() - $messages['lastupdate'];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "lastupdate" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)